addons/Request/data/settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" ?>
```

Comment:
I want it to already have those blacklisted commands that most would want to blacklist already in the settings before they add their settings to it, they can remove them if they wish, but I think having it there is better than not.